bindings/python/src/OpenSpaceToolkitPhysicsPy/Coordinate/Transform.cpp

```diff
@@ -8,9 +8,9 @@ inline void OpenSpaceToolkitPhysicsPy_Coordinate_Transform(pybind11::module& aMo
 
     using ostk::core::types::Real;
 
-    using ostk::math::obj::Vector3d;
-    using ostk::math::geom::d3::trf::rot::Quaternion;
-    using ostk::math::geom::d3::trf::rot::RotationVector;
+    using ostk::math::object::Vector3d;
+    using ostk::math::geometry::d3::transformations::rotations::Quaternion;
```

Comment:
Given that we use `object` and not `objects` in physics and math, should we use `transformation` and `rotation` ?

Reply:
I agree with this ^

Reply:
Yeah that's a fair point. I can make an MR real quick.

Reply:
The Python versions were already named with plural `transformations`/`rotations`, so I just stuck with that. But I agree that singular sounds a bit nicer. No problem with changing it!

Reply:
https://github.com/open-space-collective/open-space-toolkit-mathematics/pull/102

Reply:
👍